ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>You should use the <code>model.delete(sync=sync)</code>  method instead of <code>gcloud</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Of course 🤦 .